components/brave_new_tab_ui/components/default/rewards/style.ts

```diff
@@ -0,0 +1,150 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+import palette from 'brave-ui/theme/colors'
+
+export const WidgetWrapper = styled<{}, 'div'>('div')`
+  color: white;
+  padding: 10px 15px;
+  border-radius: 6px;
+  position: relative;
+  font-family: Muli, sans-serif;
+  overflow-x: hidden;
+  background-image: linear-gradient(140deg, #392DD1 0%, #8E2995 100%);
+`
+
+export const Footer = styled<{}, 'div'>('div')`
+  max-width: 275px;
+  margin-top: 10px;
+`
+
+export const BatIcon = styled<{}, 'div'>('div')`
+  width: 30px;
+  height: 30px;
+  display: inline-block;
+`
+
+export const RewardsTitle = styled<{}, 'span'>('span')`
+  top: -9px;
+  left: 8px;
+  font-size: 16px;
+  font-weight: 600;
+  position: relative;
+`
+
+export const ServiceText = styled<{}, 'span'>('span')`
+  color: rgba(255, 255, 255, 0.65);
```

Comment:
Did the designers mean to just change the shade of this color to a lighter one, instead of making it actually transparent to the background?

Reply:
Using white now